VOTable.tex

```diff
@@ -409,16 +396,30 @@ \subsection{Primitives}
 part of $(b+7)/8$ bytes).  These primitives
 are described in more detail in \Aref{sec:datatypes}.
 
-VOTables support two kinds of characters: ASCII 1-byte characters
-and Unicode (UCS-2) 2-byte characters. Unicode is a way to represent
-characters that is an alternative to ASCII. It uses two bytes per
-character instead of one, it is strongly supported by XML tools, and
-it can handle a large variety of international alphabets. Therefore
-VOTable supports not only ASCII strings ({\attrval{datatype}{char}}),
-but also Unicode ({\attrval{datatype}{unicodeChar}}).
+Character and string data should be encoded using the \literalvalue{char}
+type, which from VOTable 1.6 supports Unicode.
+Note that the primitive size of one byte refers to a single
+UTF-8-encoded byte, not to a single character.
+Since UTF-8 is a variable-width encoding,
+a character may require multiple bytes, and for arrays the
+string length (length in characters) and primitive count (length in bytes)
+will in general differ.
+7-bit ASCII characters are however all encoded as a single byte in UTF-8,
+so in the case of ASCII characters, which were required for this
+datatype in earlier VOTable versions, the primitive and character count
+are equal.
```

Comment:
Perhaps I'm overlooking it, and it's already there, but I think it might be worth an explicit statement in the text that clarifies that a bare `char` without a length (a one-octet string) is limited to being able to store an ASCII character.

Reply:
I've added a sentence at 73bfd135 clarifying this.  @fxpineau made a similar suggestion.